advisories/python/PSF-2025-3.json

```
@@ -24,6 +24,9 @@
             },
             {
               "fixed": "9148b77e0af91cdacaa7fe3dfac09635c3fe9a74"
+            },
+            {
+              "fixed": "a4ef689ce670684ec132204b1cd03720c8e0a03d"
```

It has been backported to 3.9 too.

```suggestion
            {
              "fixed": "a4ef689ce670684ec132204b1cd03720c8e0a03d"
            },
            {
              "fixed": "d4df3c55e4c5513947f907f24766b34d2ae8c090"
```